cx_Freeze/freezer.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         binPathExcludes: Optional[List] = None,
         includeFiles: Optional[List] = None,
         zipIncludes: Optional[List] = None,
-        silent: bool = False,
+        silentLevel: int = 0,
```

Comment:
You can maintain the variable name 'silent' in freeze.py

Reply:
You're so fast. ;-)
but I need to accept bool or int because there are users that use the API

Reply:
Just happen to be my computer.  I think I fixed the Freezer parameter now.